k8s/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE

```

Comment:
Hi @Xtigyro !

Are you attacked to GPL for this work? We tend to license everything apache2 and will need to get approval from a few internal folks if you want the k8s work licensed with GPL.

Reply:
@underscorgan So be it - I regard GPL v2 as the best licence out there but Apache 2 is fine, too. Will update it and will also put one more change in.

Reply:
Great, thank you!

Reply:
@underscorgan Done!